src/test/java/com/at/stepdefinitions/Get.java

```diff
@@ -1,51 +1,142 @@
 package com.at.stepdefinitions;
 
-import com.at.globalclasses.ApiPaths;
-import com.at.globalclasses.ApiTools;
-import com.at.globalclasses.BasicSecurityUtil;
+import com.at.globalclasses.*;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import gherkin.deps.com.google.gson.JsonParser;
 import org.junit.Assert;
 
 public class Get {
     private BasicSecurityUtil base;
+    private BasicSecurityUtil basedb;
+    long numberUsers;
 
-    public Get(BasicSecurityUtil base){
-        this.base=base;
+    public Get(BasicSecurityUtil base, BasicSecurityUtil basedb) {
+        this.base = base;
+        this.basedb = basedb;
     }
 
-    @Given("I am working on {string} environment")
-    public void i_am_working_on_environment(String env) throws Exception{
-        base.environment=env;
+
+    @Given("I am working on {string} environment")   //ready
+    public void i_am_working_on_environment(String env) throws Exception {
+        base.environment = env;
+        basedb.environment = env;
     }
 
 
-    @Given("I am targeting {string} service")
-    public void i_am_targeting_service(String service) throws Exception{
-        base.ServiceApi = new ApiTools(base.environment,service);
+    @Given("I am targeting {string} service")    //ready
+    public void i_am_targeting_service(String service) throws Exception {
+        base.ServiceApi = new ApiTools(base.environment, service);
     }
 
-    @Given("I want to retrieve all users")
-    public void i_want_to_retrieve_all_users() throws Exception{
-        base.apiResource= ApiPaths.SCE_GET_USERS;
+    @Given("I have access to {string} database")   //ready
+    public void iHaveAccessToDatabase(String dataBase) {
+        basedb.uridb = dataBase;
+
     }
 
-    @When("I send a GET request")
+    @Given("I want to retrieve all users")  //ready
+    public void i_want_to_retrieve_all_users() throws Exception {
+        base.apiResource = ApiPaths.SCE_GET_USERS;
+    }
+
+    @When("I send a GET request")   //ready
```

Comment:
Please remove comments that are not longer used.

Reply:
ready.

Reply:
Done.